src/core/server/ui_settings/ui_settings_client.test.ts

```diff
@@ -624,6 +624,22 @@ describe('ui settings', () => {
       expect(await uiSettings.get('dateFormat')).toBe('foo');
     });
 
+    it('returns the overridden value for key theme:version', async () => {
+      const opensearchDocSource = { 'theme:version': 'v8 (beta)' };
+      const overrides = { 'theme:version': 'v7' };
+      const { uiSettings } = setup({ opensearchDocSource, overrides });
```

Comment:
what is `overrides` doing in this context? Do we need another test to validate that `v8 (beta)` versions are re-written even without `overrides`?

Reply:
Here `opensearchDocSource` already has a saved object schema which has `theme:version` set to `v8`. When we set the config property to new value in `overrides`, the new value is read as part of ui settings and the value stored in the _doc/source is ignored. 
I've added the new test for the use case when there is no override.